packages/transport/src/types/messages.ts

```diff
@@ -1257,11 +1257,40 @@ export type EosSignedTx = {
     signature: string;
 };
 
+export enum EthereumDefinitionType {
+    NETWORK = 0,
+    TOKEN = 1,
+}
+
+// EthereumNetworkInfo
+export type EthereumNetworkInfo = {
+    chain_id: number;
+    symbol: string;
+    slip44: number;
+    name: string;
+};
+
+// EthereumTokenInfo
+export type EthereumTokenInfo = {
+    address: string;
+    chain_id: number;
+    symbol: string;
+    decimals: number;
+    name: string;
+};
+
+// EthereumDefinitions
+export type EthereumDefinitions = {
+    encoded_network?: ArrayBuffer;
+    encoded_token?: ArrayBuffer;
+};
```

Comment:
When generating it with `./scripts/protobuf-build.sh`  the result is:

```
  encoded_network?: string;
  encoded_token?: string;
```

But in reality `ArrayBuffer` is the right one that we are using. But if this file is updated automatically wouldn't that cause problem in the future if we change it to `ArrayBuffer` now?

Reply:
really? did I change it manually?  I don't remember. But you can check
protobuf patches we use them when we want to somehow modify types generation
https://github.dev/trezor/trezor-suite/blob/2836861d877dd46c06fcc966190cf1bf5d5defed/packages/transport/scripts/protobuf-patches/index.js#L13
and regarding buffers treated as strings, there is support for that here https://github.dev/trezor/trezor-suite/blob/2836861d877dd46c06fcc966190cf1bf5d5defed/packages/transport/src/lowlevel/protobuf/encode.ts#L7